app/helpers/likes_helper.rb

```diff
@@ -0,0 +1,2 @@
+module LikesHelper
+end
```

Comment:
Can prob delete this

Reply:
Code broke without this so will have to keep it :3